src/components/SwapWidget/SwapWidget.tsx

```diff
@@ -604,14 +620,7 @@ const SwapWidget: FC<SwapWidgetPropsType> = ({
         dispatch(clearTradeTermsQuoteAmount());
       }
 
-      if (e.code && e.code === 4001) {
```

Comment:
This didn't work anymore. Because the error gets "rethrown" the `code` gets moved into the `stack`.